examples/pubsub/pub/pub.go

```diff
@@ -44,7 +44,7 @@ func main() {
 	}
 
 	// Publish multiple events
-	if res := client.PublishEvents(ctx, pubsubName, topicName, publishEventsData); res.Error != nil {
+	if res := client.PublishEvents(ctx, pubsubName, bulkTopicName, publishEventsData); res.Error != nil {
 		panic(err)
```

Comment:
This doesn't seem entirely right, you're panicking here with an unrelated (potentially) nil error